src/main/java/com/rocketden/main/exception/GameError.java

```diff
@@ -10,10 +10,11 @@
 
     BAD_LANGUAGE(HttpStatus.BAD_REQUEST, "An invalid language was chosen."),
     BAD_SETTING(HttpStatus.BAD_REQUEST, "An invalid game setting was provided."),
-    USER_NOT_IN_GAME(HttpStatus.BAD_REQUEST, "The requested action could not be completed because the user is not in the game."),
-    INVALID_PERMISSIONS(HttpStatus.FORBIDDEN, "You do not have permission to perform this action."),
     EMPTY_FIELD(HttpStatus.BAD_REQUEST, "Please ensure a value is provided for each required field."),
-    NOT_FOUND(HttpStatus.NOT_FOUND, "A game could not be found with the given id.");
+    GAME_NOT_OVER(HttpStatus.FORBIDDEN, "You may not perform this action because the game is not over."),
+    INVALID_PERMISSIONS(HttpStatus.FORBIDDEN, "You do not have permission to perform this action."),
+    NOT_FOUND(HttpStatus.NOT_FOUND, "A game could not be found with the given id."),
+    USER_NOT_IN_GAME(HttpStatus.BAD_REQUEST, "The requested action could not be completed because the user is not in the game.");
```

Comment:
Did you change these to arrange alphabetically? Should I keep that system on future enums?

Reply:
Yeah - I figured just whenever we get around to adding/modifying the errors, we can just reorder them to be in alphabetical order (or maybe order them by HTTP status might be better, either way)